onnx/onnx.proto

```diff
@@ -64,15 +65,25 @@ message AttributeProto {
   repeated GraphProto graphs = 11;    // list of graph
 }
 
+// Defines information on value, including the name, the type, and
+// the shape of the value.
+message ValueInfoProto {
+  // This field MUST be present in this version of the IR.  
+  optional string name = 1;     // namespace Value
+  // This field MUST be present in this version of the IR.  
```

Comment:
I think it MUST be present only if it's graph inputs/outputs. Let's clarify that if it's used as a part of internal values definitions (`value_info` below) then it's fine to have type optional

Reply:
It is optional for a value to appear in `value_info`. So when a value appears in `value_info`, the PR currently does require type.  Do you think we should relax it further in this case?